src/opencl_rawsha512_fmt_plug.c

```diff
@@ -307,7 +307,7 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 	global_work_size = GET_NEXT_MULTIPLE(count, local_work_size);
 
 	///Copy data to GPU memory
-	if (sha512_key_changed || ocl_autotune_running) {
+	if (sha512_key_changed) {
```

Comment:
Why don't you rename `sha512_key_changed`?

Reply:
I simply overlooked that one